.appveyor.yml

```diff
@@ -41,6 +42,9 @@ test_script:
       "Running npm run test:without-system-tests..."
       "--------------------------------------------"
       npm run test:without-system-tests
+      npm run coverage:system-tests
+      npm run aggregateJUnit
+      junit-merge -d junit-aggregate -o junit-aggregate.xml
```

Comment:
Using junit-merge to merge all results into one file. There's an issue when it does it recursively so I aggregate them all into a single directory with the aggregateJUnit script.